srcpkgs/tiny-irc-client/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'tiny-irc-client'
+pkgname=tiny-irc-client
```

Comment:
Package name should be `tiny`, also update commit message to reflect name change.

Reply:
ok

Reply:
Once you change it to 'tiny' you wont need the wrksrc* lines

Reply:
yes i noticed that, thanks, but i still need build_wrksrc right ?